File: @LV1/1.7/ostgotska/ostgotska.hs
import Data.List

main :: IO ()
main = do
  s <- words <$> getLine
  let ae = filter (isInfixOf "ae") s

  if 10 * length ae >= 4 * length s
    then putStrLn "dae ae ju traeligt va"
    else putStrLn "haer talar vi rikssvenska"

File: @LV1/1.7/ostgotska/ostgotska.cpp
#include <cstdio>
#include <cstring>

int main() {
    char tmp[100];
    int s = 0, ae = 0;
    while (scanf("%s", tmp) == 1) {
        s++;
        if (strstr(tmp, "ae") != NULL) {
            ae++;
        }
    }
    if (10 * ae >= 4 * s) {
        puts("dae ae ju traeligt va");
    } else {
        puts("haer talar vi rikssvenska");
    }
}